src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/services/services.extensions.ts

```diff
@@ -63,6 +64,12 @@ class ShareContentDetail extends generated.ShareContentDetail {
   }
 }
 
+class SchemaDetail extends generated.SchemaDetail {
+  get filteredLayerSchemaIds(): string[] | undefined {
```

Comment:
Find a better name and add jsdoc comments. e.g. nonSystemLayerSchemaIds

Reply:
Removed the property as it is not used